src/visitpy/common/MethodDoc.C

```diff
@@ -3608,6 +3608,36 @@ const char *visit_GetLastError_doc =
 "# Get last message into msg and then clear last error message to \"\"\n"
 "msg = GetLastError(1)\n"
 ;
+const char *visit_GetLastMessage_doc = 
```

Comment:
I am running into weirdness trying to update CLI docs.

I don't fully understand what I am seeing in the history of `MethodDoc.C`but it looks like this was a direct change to `MethodDoc.C` instead of via `functions.rst` and then regenerating `MethodDoc.C`

Reply:
Brad was probably not aware of this part of the process and I didn't remember either.
Sorry for this problem.